src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -636,11 +639,11 @@ const FeatureFlagsPage: React.FC = () => {
         refetch()
       }}
     >
-      {hasFeatureFlags && (
-        <Container padding={{ top: 'medium', right: 'xlarge', left: 'xlarge' }}>
-          {FILTER_FEATURE_FLAGS && (
-            <FlagTableFilters features={features} currentFilter={flagFilter} updateTableFilter={setFlagFilter} />
-          )}
+      <Container width="100%" height="100%" padding={{ top: 'medium', right: 'xlarge', left: 'xlarge' }}>
```

Comment:
Since `Container` is just a `div`, it'll be full width already
```suggestion
      <Container height="100%" padding={{ top: 'medium', right: 'xlarge', left: 'xlarge' }}>
```